crates/sui-analytics-indexer/src/package_store.rs

```diff
@@ -32,7 +32,7 @@ impl From<Error> for PackageResolverError {
         match source {
             Error::TypedStore(store_error) => Self::Store {
                 store: STORE,
-                source: Box::new(store_error),
+                source: Arc::new(store_error),
```

Comment:
Switch from `Box` to `Arc` was required because the `DataLoader` API requires the `Error` type to have `Clone` and `Box<dyn Error>` does not have that.